web/src/components/IconWithUiDirection.tsx

```diff
@@ -0,0 +1,12 @@
+import styled, { css } from 'styled-components'
+
+import { UiDirectionType } from 'translations/src'
+
+export const IconWithUiDirection = styled.img<{ direction?: UiDirectionType }>`
```

Comment:
Remove named import

Reply:
`import { UiDirectionType } from 'translations'`

Reply:
What do you mean by that comment? I was referring to the following:
```suggestion
const IconWithUiDirection = styled.img<{ direction?: UiDirectionType }>`

export default IconWithUiDirection
```

Reply:
fixed in last commit